cypress/support/step-definitions/common-steps.js

```diff
@@ -138,6 +142,14 @@ Given('I open basic Design System {string} component docs page', (component) =>
   visitDocsUrl(component, 'basic', false, 'design-system-');
 });
 
+Given('I open default Design System {string} component docs page', (component) => {
+  visitDocsUrl(component, 'default_story', false, 'design-system-');
+});
+
+Given('I open default Design System {string} component canvas page', (component) => {
```

Comment:
And delete this step, if it uses `visitComponentUrl`  instead of `visitDocsUrl`

Reply:
deleted